src/Loading/Loading.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
```

Comment:
I suggest additionally importing `{ ElementType, FC }` (and simplifying the uses below) since at some point we're going to remove the base `React` import.

Reply:
I'd been adding in React imports because jest tests fail without it, but I think I just need to configure babel a bit more. I'll finish the automatic jsx transform and then remove our React imports.

Reply:
Done.